Gemfile

```diff
@@ -1,7 +1,7 @@
 source 'https://rubygems.org'
 git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
-ruby '3.2.2'
+ruby '3.2.3'
```

Comment:
Have we considered reading this from either the `.ruby-version` or `.tool-versions` file?
https://blog.saeloun.com/2024/01/17/rubygems-introduced-a-file-option-to-specify-ruby-version/#:~:text=Rubygems%20introduced%20a%20%3Afile%20option,will%20automatically%20read%20the%20version.&text=NOTE%3A%20Ensure%20you're%20on%20bundler%202.5.